src/oauth.ts

```diff
@@ -13,14 +24,50 @@ interface IParams
     use_https: boolean
 }
 
-interface request {
-    url: string
+/**
+ * {
+    "accessToken": "",
+    "expiresIn": 14405,
+    "refreshToken": "",
+    "scope": [
+        "channel:moderate",
+        "chat:edit",
+        "chat:read"
+    ],
+    "tokenType": "bearer"
+}
+ */
+interface IResponse 
+{
+    access_token: string,
+    expires_in: number,
+    refresh_token: string,
+    scope: string[],
+    token_type: string;
 }
 
-interface response {
-    statusCode: number,
-    write: Function,
-    end: Function
+/**
+ * @description Change our response from snake case to camel case.
+ * @param res 
+ * @returns 
+ */
+function fixResponse(res: IResponse): { accessToken: string, expiresIn: number, refreshToken: string, scope: string[], tokenType: string }
+{
+    const hydr: IExportOBJ = hydra(res) as IExportOBJ;
```

Comment:
We have to change it to camel case from snake case or else Twurple will throw an error. It only supports camel case; also tried to throw in Hydra because that's just a quality lib. (not biased)

Reply:
Implemented [this](https://discord.com/channels/993857770233278474/998098626335219742/1016856237725466674) as you said; it works, on you if it has bad performance. <3